test/e2e/data/infrastructure-provider-byoh/v1beta1/templates/docker/cluster-template-topology-docker.yaml

```diff
@@ -13,7 +13,7 @@ spec:
         - 192.168.0.0/16
     serviceDomain: cluster.local
   topology:
-    class: quickstart
+    class: quickstart-docker
```

Comment:
@shivi28 I assume this will be converted to a variable, right?

Reply:
Yes, with this [issue](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/issues/643) we can make it variable. I will do it before the upstream release.